akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/TestProbeSpec.scala

```diff
@@ -37,7 +39,7 @@ class TestProbeSpec extends ScalaTestWithActorTestKit with WordSpecLike {
     "allow probing for actor stop when actor already stopped" in {
       val probe = TestProbe()
       val ref = spawn(Behaviors.stopped)
-      probe.expectTerminated(ref, 100.millis)
+      probe.expectTerminated(ref, shortDuration)
```

Comment:
This was the failing test. I see no reason to use a short duration at all here. `probe.expectTerminated(ref)` should do it.

Reply:
:+1:

Reply:
The other timeouts were tight so i'll leave the other changes in